hw1_oop/HW1_python2_OOP.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+
+class Chat:
+    default_start = datetime(1970, 1, 1)
+    
+    def __init__(self):
+        self._chat_history = []    # List of all messages in chat
+            
+    def show_last_message(self):
+        last_message = self._chat_history[0]
+        print(last_message.show())
+    
+    def get_history_from_time_period(self, start=None, end=None):
+        if start is None:
+            start = self.default_start
+        if end is None:
+            end = datetime.now()
```

Comment:
Я бы лучше использовал время первого и последних сообщений в качестве дефолтных дат, тем более, что их просто получить

```suggestion
        if start is None:
            start = self._chat_history[-1].datetime
        if end is None:
            end = self._chat_history[0].datetime
```